onnxruntime/core/providers/js/js_execution_provider.cc

```diff
@@ -500,6 +502,8 @@ std::unique_ptr<KernelRegistry> RegisterKernels() {
       BuildKernelCreateInfo<ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSInternalNHWCDomain, 1, float, GlobalAveragePool)>,
       BuildKernelCreateInfo<ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSInternalNHWCDomain, 1, float, GlobalMaxPool)>,
 
+      BuildKernelCreateInfo<ONNX_CPU_OPERATOR_TYPED_MS_KERNEL(kJsExecutionProvider, kMSDomain, 1, float, FusedConv)>,
```

Comment:
Specifying the domain, kMSDomain, is not required when using ONNX_CPU_OPERATOR_TYPED_MS_KERNEL macro. Only name, version, type, builder, etc. are required.